docker/README.md

```diff
@@ -43,6 +43,32 @@ Once this is done you should run the following command to build `aarch64` static
 ./docker-static-build.sh aarch64
 ```
 
+### Packages from statically linked binaries
+
+It's possible to create packages (ubuntu/debian & fedora) using already
+built binaries. This means that the build step of the package will be reduced
+to copying the binaries instead of building again from scratch.
+
+Using Ubuntu/Debian and assuming all Tezos binaries are located in the `out` directory:
+```sh
+# DEB_BUILD_OPTIONS=nostrip is needed if the provided binaries have already
+# been stripped of any debug symbols.
+
+docker build -t tezos-ubuntu -f docker/package/Dockerfile-ubuntu .
+docker run -it -e TEZOS_VERSION=v13.0 \
+               -e DEB_BUILD_OPTIONS=nostrip \
+               -v `pwd`:/tezos-packaging tezos-ubuntu \
```

Comment:
I slightly dislike the idea of mounting the whole directory as a volume because build artifacts and intermediate build directories pollute it. Does it make sense to mount only the directory with the binaries and the directory with the build artifacts?

Reply:
This will also probably help to solve the issues I've mentioned above

Reply:
Yep, I've updated the example.